pilot/cmd/pilot-agent/health.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+
+	"context"
+	"sync"
+
+	"istio.io/istio/pilot/pkg/proxy/envoy"
+	"istio.io/istio/pkg/log"
+)
+
+const (
+	healthzPath = "/healthz/"
```

Comment:
should this just be `/healthz`?

Reply:
yes.

Reply:
Actually, as described in [the health checking doc](https://docs.google.com/document/d/1RxO3IBT8z68XNmM8tGWxrUcSbrRneiBWlL1A4qZf3CU/edit#heading=h.x9snb54sjlu9), it should be `/healthz/ready`.